### PR TITLE
Closes: Issue #3145 curl.ts example with reference in manual

### PR DESCRIPTION
**Pull request by hartror**
Adds a simple curl example with a reference in the manual.

Closes: https://github.com/denoland/deno/issues/3145

### CODE REVIEW

File: std/examples/curl.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+async function curl(url: string): Promise<void> {
+  const res = await fetch(url);
+  await Deno.copy(Deno.stdout, res.body);
+}
```
Comment (nayeemrmn) [inline std/examples/curl.ts]: Make this return a `Reader` and export it. The copy can happen below.

File: std/examples/curl.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+await curl(Deno.args[1]);
+Deno.exit(0);
```
Comment (nayeemrmn) [inline std/examples/curl.ts]: Wrap this in an `import.meta.main` check.

File: std/examples/curl.ts (hunk 1 above)
Comment (hartror) [reply std/examples/curl.ts]: Yeah I cribbed this from `std/examples/cat.ts`.

So no changes required?

Comment (nayeemrmn) [reply std/examples/curl.ts]: No, you can resolve this thread.

Comment (hartror) [reply std/examples/curl.ts]: Thanks

**Pull request merged**